website/nodemon.json

```diff
@@ -2,5 +2,5 @@
   "watch": ["."],
   "ext": "ts",
   "ignore": ["dist", "pages", "out", "public", "custom-generated-pages"],
-  "exec": "cross-env NODE_ENV=development ts-node --project tsconfig.server.json server.ts"
+  "exec": "cross-env NODE_OPTIONS='--inspect' NODE_ENV=development ts-node --project tsconfig.server.json server.ts"
```

Comment:
Out of curiosity, what does inspect flag does?

Reply:
generally it allows to attach node.js debugger